traffic_ops/app/lib/API/User.pm

```diff
@@ -281,11 +284,14 @@ sub current {
 
 	if ( &is_ldap($self) ) {
 		my $role = $self->db->resultset('Role')->search( { name => "read-only" } )->get_column('id')->single;
+		my $user_tenant_id = $self->current_user_tenant();
```

Comment:
just so you know. $self->current_user_tenant() will alway return undef in this case. if you are is_ldap, that means you have no entry in the tm_user table...so in my opinion, you might just want to hardcode this to undef and save the unnecessary query.

"tenantId"	  => undef,
 "tenant"          => undef,

go take a look at is_ldap to see what i mean: https://github.com/apache/incubator-trafficcontrol/blob/master/traffic_ops/app/lib/UI/Utils.pm#L257